cookiecutter.json

```diff
@@ -43,6 +44,10 @@
     ],
     "gitops_project": "{% if cookiecutter.deployment_strategy == 'gitops' %}{{ cookiecutter.project_slug }}-gitops{% else %}(none){% endif %}",
     "docker_registry": "{{ cookiecutter.cloud_platform.replace('APPUiO', 'registry.appuio.ch').replace('(none)', cookiecutter.vcs_platform.replace('Bitbucket.org', 'hub.docker.com').replace('GitHub.com', 'quay.io').replace('GitLab.com', 'registry.gitlab.com') + '/' + cookiecutter.vcs_account) }}",
+    "docker_image": "{{ cookiecutter.project_slug }}",
+    "registry_user": "{{ cookiecutter.ci_service.replace('.yml', '').replace('.', '').replace('-steps', '').replace('travis', 'travis-ci') }}",
+    "automation_user": "{{ cookiecutter.ci_service.replace('.yml', '').replace('.', '').replace('-steps', '').replace('travis', 'travis-ci') }}",
+    "flux_ssh_known_hosts": "",
```

Comment:
@ZuGao This is not in use yet. I'll remove it for this PR to make sense to be merged.

You can still add it later, but I'd suggest to use a more generic naming (something with "gitops" maybe) or have this added in some other way.